CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.10.1 (2021-03-15)
+
+- Fixed the timepicker component to initialize the control value during lifecycle hook. [#253](https://github.com/blackbaud/skyux-datetime/pull/253)
```

Comment:
```suggestion
- Fixed the timepicker component so that it initializes the control value during lifecycle hook. [#253](https://github.com/blackbaud/skyux-datetime/pull/253)
```